### PR TITLE
Create email service interface on domain (Jira: ATQR-81 )

### PR DESCRIPTION
**Pull request by albnunes**
I organized the files, created and tested the email service

Linear: ATQR-136

### CODE REVIEW

File: libs/atqr/domain/src/lib/valueObjects/email.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { ValueObject } from '../../utils/valueObject';
```
Comment (rairezende) [inline libs/atqr/domain/src/lib/valueObjects/email.ts]: shouldn't this be a scope path instead of a relative one?

**Review by rairezende — CHANGES_REQUESTED** [review]

File: libs/atqr/domain/src/lib/valueObjects/email.ts (hunk 1 above)
Comment (albnunes) [reply libs/atqr/domain/src/lib/valueObjects/email.ts]: resolved

File: libs/atqr/domain/src/lib/valueObjects/index.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
-export * from './contact' 
+export * from './contact'
 export * from './email'
+export * from './emailAddress'
 export * from './invitee'
-export * from './enrollment'
```
Comment (rairezende) [inline libs/atqr/domain/src/lib/valueObjects/index.ts:4]: why did you take enrollment and officiation value objects imports out of here?

File: libs/atqr/domain/src/lib/EmailService/interfaces/index.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export * from './email.interface';
+export * from './mailer.interface';
```
Comment (rairezende) [inline libs/atqr/domain/src/lib/EmailService/interfaces/index.ts]: You're exporting this "mailer interface", but I don't see you using it. Is it supposed to be used eventually?

File: libs/atqr/domain/src/lib/valueObjects/index.ts (hunk 2 above)
Comment (albnunes) [reply libs/atqr/domain/src/lib/valueObjects/index.ts:4]: anywhere they are needed they can be imported by "@atqr/domain"

Comment (rairezende) [reply libs/atqr/domain/src/lib/valueObjects/index.ts:4]: I think the biggest question is why aren't they in the Value Objects folder anymore. But that isn't part of your task anyway :)

**Review by rairezende — APPROVED** [review]

**Pull request merged**